.github/ISSUE_TEMPLATE/enhancement-request.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Enhancement request
+about: Anything you’d like to see in scanpy?
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+<!-- What kind of feature would you like to request? -->
+- [ ] Additional function parameters / changed functionality / changed defaults?
+- [ ] Core functionality: Something important is missing!
+- [ ] External tools: Do you know an existing package that should go into `scanpy.external.*`?
+- [ ] Other?
```

Comment:
I think there's another couple things here that might not be covered:
`- [ ] New plotting function: A plot you would like to see, but are missing?`
`- [ ] New convenience function: A simple analysis tool you have been using and are missing in sc.tools? `

The latter would be for things like `sc.tl.marker_gene_overlap()` or the stuff in `sc.metrics`.